examples/functions/functions_tester/functions/additionWithRetryHandler.js

```diff
@@ -0,0 +1,39 @@
+// :snippet-start: addition-with-retry-handler
```

Comment:
[q] what's `:snippet-start` used for?

Reply:
it's for generating the code examples - https://mongodb-university.github.io/Bluehawk/reference/tags#snippet